server/routes/objective.test.ts

```diff
@@ -111,14 +119,22 @@ describe('GET /sentence-plan/objective', () => {
       .expect(res => expect(res.text).toContain('Select at least one criminogenic need'))
   })
 
+  it('should validate motivation', () => {
+    return request(app)
+      .post('/sentence-plan/1/objective/2')
+      .send({ description: 'New text', 'relates-to-needs': 'yes', needs: 'accommodation' })
+      .expect('Content-Type', /html/)
+      .expect(res => expect(res.text).toContain('Please select a motivation level'))
+  })
+
   it('should save data', () => {
     const updateApi = jest.fn().mockResolvedValue({})
     services.sentencePlanClient.updateObjective = updateApi
     return request(app)
       .post('/sentence-plan/1/objective/2')
-      .send({ description: 'New text', 'relates-to-needs': 'no', needs: [] })
+      .send({ description: 'New text', 'relates-to-needs': 'no', needs: [], motivation: [] })
```

Comment:
You should probably set this to a meaningful string, as we don't expect motivation to be an array e.g.
```suggestion
      .send({ description: 'New text', 'relates-to-needs': 'no', needs: [], motivation: 'Preparation' })
```